### DISCUSSION TITLE
Required data to provide for redeploy via API ?

### DISCUSSION THREAD

**Discussion by Amirthananth** [vercel/community#2719] [Help] [answered]
### Summary

Hey there,

I understand from the docs that for [Create a new deployment](https://vercel.com/docs/rest-api/endpoints#create-a-new-deployment), a large number of body data is required.

Is there a way to trigger a redeploy of a git-backed project via API with just the existing deploymentId for which we need redeployment? Also finding the required details doesn't seem to be an easy task because we are not able to retrieve the details from [Get a deployment by ID or URL](https://vercel.com/docs/rest-api/endpoints#get-a-deployment-by-id-or-url) as well

Thanks.

**Comment by smaeda-ks** [comment, marked as answer]
@Amirthananth have you tried Deploy Hook?
https://vercel.com/docs/concepts/deployments/deploy-hooks
